pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -97,7 +97,7 @@ sub _suggest_kernelcare {
     return if !Cpanel::KernelCare::system_supports_kernelcare() || Cpanel::Security::Advisor::Assessors::Symlinks->new->has_cpanel_hardened_kernel();
 
     # Abort if kernelcare is already licensend
-    return if try { Cpanel::KernelCare::Availability::system_license_from_cpanel(); };
+    return if eval { Cpanel::KernelCare::Availability::system_license_from_cpanel(); };
```

Comment:
```suggestion
    local $@;
    return if eval { Cpanel::KernelCare::Availability::system_license_from_cpanel(); };
```
Otherwise you introduce risk that some poorly-written code out there will break because $@ is now being overwritten.

Reply:
I didn't see anything downstream specifically that was using $@, but I get your point.  I'll be more careful next time.